tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -1572,18 +1565,11 @@ error: Expected type to be \\"num\\", but got \\"str\\" instead
    |                         ^^^ Expected type to be \\"num\\", but got \\"str\\" instead
 
 
-error: Expected \\"Array\\" type, found \\"MutSet<num>\\"
+error: Expected \\"Array\\" or \\"MutArray\\", found \\"MutSet<num>\\"
    --> ../../../examples/tests/invalid/mut_container_types.w:11:10
    |
 11 | let s2 = MutSet<num>[1, \\"2\\", 3];
-   |          ^^^^^^^^^^^^^^^^^^^^^^ Expected \\"Array\\" type, found \\"MutSet<num>\\"
-
-
-error: Expected type to be \\"num\\", but got \\"str\\" instead
-   --> ../../../examples/tests/invalid/mut_container_types.w:11:25
-   |
-11 | let s2 = MutSet<num>[1, \\"2\\", 3];
-   |                         ^^^ Expected type to be \\"num\\", but got \\"str\\" instead
```

Comment:
This error no longer happens, and imo I'm not sure if it should be considered a regression. 
`MutSet<num>[1, "2", 3]` is simply not a valid syntax, the fact that it worked before was kinda an accident of the implementation.

We could probably get it working intentionally, but idk how important that is.

Reply:
Looks fine to me